lighthouse-core/audits/is-on-https.js

```diff
@@ -30,6 +30,14 @@ const UIStrings = {
     }`,
   /** Label for a column in a data table; entries in the column will be the URLs of insecure (non-HTTPS) network requests. */
   columnInsecureURL: 'Insecure URL',
+  /** Label for a column in a data table; entries in the column will be how the browser handled insecure (non-HTTPS) network requests. */
+  columnResolution: 'Resolution',
+  /** Value for the resolution column in a data table; denotes that the insecure URL was blocked by the browser. */
+  blocked: 'Blocked',
+  /** Value for the resolution column in a data table; denotes that the insecure URL may be blocked by the browser in the future. */
+  warning: 'Warning',
+  /** Value for the resolution column in a data table; denotes that the insecure URL was upgraded to a secure request by the browser. */
+  upgraded: 'Upgraded',
```

Comment:
lets match the dt strings? 
https://source.chromium.org/chromium/chromium/src/+/master:third_party/devtools-frontend/src/front_end/sdk/MixedContentIssue.js;l=97-100;drc=906108beba6107af7259dff5366375f70f78267e

warning -> warned

upgraded -> automatically upgraded